lib/sycamore/sycamore/connectors/opensearch/utils.py

```diff
@@ -6,20 +6,29 @@
 
 
 @context_params("opensearch")
-def get_knn_query(text_embedder: Embedder, query_phrase: str, k: int = 500, context: Optional[Context] = None):
+def get_knn_query(
```

Comment:
Nit: Why not throw in `max_distance` while you're at it so all three radial search options are available?

Reply:
Because it only supports a min_score lol. There is a max_distance param that expects an actual physical distance value, skipping that because it's just going to cause confusion until we have a use case

https://opensearch.org/docs/latest/search-plugins/knn/radial-search-knn/#supported-cases